servers/republik/modules/crowdfundings/graphql/resolvers/User.js

```diff
@@ -113,12 +114,19 @@ module.exports = {
       memberships = await resolveMemberships({ memberships, pgdb })
 
       const activeMembership = memberships.find(m => m.active)
-      if (
-        activeMembership &&
-        activeMembership.membershipType.name === 'ABO_GIVE_MONTHS'
-      ) {
-        debug('active membership type "ABO_GIVE_MONTHS", return prolongBeforeDate: null')
-        return null
+      if (activeMembership) {
+        if (activeMembership.membershipType.name === 'ABO_GIVE_MONTHS') {
+          debug('active membership type "ABO_GIVE_MONTHS", return prolongBeforeDate: null')
+          return null
+        }
+
+        if (!ignoreAutoPayFlag) {
+          const autoPay = await autoPaySuggest(activeMembership.id, pgdb)
+          if (autoPay) {
```

Comment:
The not [overdue part (`lastEndDate > moment().subtract(1, 'day')`)](https://github.com/orbiting/backends/pull/472/files#diff-cee2cfea33e6969e45a1d051cdc6c0e3L155) is missing. We should keep that because overdue auto pay customer may have missed the emails and then get a visual reminder on the site and the possibility to also switch to a different card or payment method.

Reply:
Indeed.